blocks/header/header.js

```diff
@@ -491,6 +491,52 @@ function handleScrolling(el) {
   });
 }
 
+function handleLoginButtonEvent() {
+  // get all login buttons (nav, header, homepage-header)
+  const loginButtons = document.querySelectorAll('#login-button');
```

Comment:
You're not supposed to have multiple ids with the same value on the page especially for forms see https://www.chromium.org/developers/design-documents/create-amazing-password-forms/#follow-html-guidelines. You can use a classname instead.